### DISCUSSION TITLE
Allow setting time zone for cron jobs

### DISCUSSION THREAD

**Discussion by adamcstephens** [woodpecker-ci/woodpecker#2262] [Request a Feature] [answered]
### Clear and concise description of the problem

As a user of cron jobs, I would like to have a consistent local time that my jobs are run. Currently cron jobs are set in UTC but my time zone like many others has a daylight saving time change. This means that the time I configure a cron job for now, will be an hour different during ~half the year.

### Suggested solution

The user facing changes for this could be a configuration option for administrators to set the time zone for cron jobs, or the ability to set the time zone when configuring a cron job. 

### Alternative

I've considered manually changing all the cron jobs twice a year, but this is tedious and error prone.

### Validations

- [X] Checked that the feature isn't part of the `next` version already [https://woodpecker-ci.org/faq#which-version-of-woodpecker-should-i-use]
- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't already an [issue](https://github.com/woodpecker-ci/woodpecker/issues) that request the same feature to avoid creating a duplicate.

**Comment by 6543** [comment]
https://github.com/woodpecker-ci/woodpecker/blob/cfdb32ae45c9b1dd4af66c152de4fc41a89f7bf5/server/cron/cron.go#L71

**Reply by plague-doctor** [reply]
Can we have the local time for cron? Fiddling with UTC is... unpleasant.

**Comment by 6543** [comment, marked as answer]
new release will have it -> https://github.com/woodpecker-ci/woodpecker/pull/6597
